_DeploymentAndDistroPackaging/ansible/group_vars/all

```diff
@@ -1,4 +1,5 @@
 ---
+workdir: /tmp/ciao
```

Comment:
http://docs.ansible.com/ansible/file_module.html says the mode is optional and doesn't say what happens if it is not given.